wdl/tasks/hotspot1.wdl

```diff
@@ -0,0 +1,39 @@
+version 1.0
+
+
+import "../structs/hotspot1.wdl"
+import "../structs/resources.wdl"
+
+
+task runhotspot {
+    input {
+        File subsampled_bam
+        HotSpot1Index index
+        HotSpot1Params params
+        Resources resources
+    }
+
+    String prefix = basename(subsampled_bam, ".bam")
+
+    command {
+        ln ~{index.chrom_info} .
+        ln ~{index.mappable_regions} .
+        runhotspot.bash \
+            $HOTSPOT_DIRECTORY \
+            $PWD \
```

Comment:
I think we could get away with hardcoding this as `.`

Reply:
Yeah the `.` was causing a lot of problems with underlying script. It must hardcode the `.` and change contexts or something.

Reply:
nevermind, this is good as it is.